src/babel-preset-adeira/src/__tests__/__snapshots__/configs.test.js.snap

```diff
@@ -276,7 +370,62 @@ exports[`emits correct config for target 'js-esm' and environment '{"node":"curr
         "bugfixes": true
       }
     ],
-    "@babel/preset-react"
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "classic"
+      }
+    ]
+  ],
+  "plugins": [
+    "<BABEL_PRESET_ROOT>/src/dev-expression-check.js",
+    "<BABEL_PRESET_ROOT>/src/dev-expression.js",
+    "<BABEL_PRESET_ROOT>/src/adeira-js-invariant.js",
+    "<BABEL_PRESET_ROOT>/src/adeira-js-warning.js",
+    "@babel/plugin-transform-flow-strip-types",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-numeric-separator",
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "useESModules": true
+      }
+    ]
+  ],
+  "parserOpts": {
+    "plugins": [
+      "jsx",
+      "flow",
+      "flowComments",
+      "bigInt",
+      "throwExpressions"
+    ]
+  },
+  "retainLines": false
+}
+`;
+
+exports[`emits correct config for target 'js-esm' and environment '{"reactRuntime":"automatic"}' 1`] = `
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "debug": false,
+        "modules": false,
+        "targets": {
+          "reactRuntime": "automatic"
+        },
+        "bugfixes": true
+      }
+    ],
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "classic"
```

Comment:
What's going on here? 🤔 The title says `emits correct config for target 'js-esm' and environment '{"reactRuntime":"automatic"}'` but the payload is a bit fishy (looks the mixed values automatic/classic).

Reply:
Yeah, looks like I misunderstood the test setup. I will test it differently

Reply:
There is a test matrix that tests every config combination. To motivation was to test every possible combination of the config we can have. Seems like you extended the wrong input there, right (environment instead of a new option)?

Reply:
Yes, exactly, I am fixing it now 😊

Reply:
Please check again 🙏